### ISSUE TITLE
org.elasticsearch:elasticsearch:7.11.0 depends on unpublished elasticsearch-plugin-classloader

### ISSUE THREAD

**Issue by dw-alex** [elastic/elasticsearch#68888]
The [org.elasticsearch:elasticsearch:7.11.0 pom on maven central](https://repo1.maven.org/maven2/org/elasticsearch/elasticsearch/7.11.0/elasticsearch-7.11.0.pom) depends on a package that does not exist on maven central (https://repo1.maven.org/maven2/org/elasticsearch/elasticsearch-plugin-classloader returns a 404).

```
    <dependency>
      <groupId>org.elasticsearch</groupId>
      <artifactId>elasticsearch-plugin-classloader</artifactId>
      <version>7.11.0</version>
      <scope>runtime</scope>
    </dependency>
```

This leads to failures in our build.

**Comment by alexklibisz** [comment]
For now you can run the following to get a locally-cached copy of the module:
```
#!/bin/bash
cd /tmp
wget --quiet -O v7.11.0.zip https://github.com/elastic/elasticsearch/archive/v7.11.0.zip
rm -rf elasticsearch-7.11.0
unzip -q v7.11.0.zip
cd elasticsearch-7.11.0
./gradlew -Dbuild.snapshot=false :libs:elasticsearch-plugin-classloader:publishToMavenLocal
```

The release zip is about 13mb and the whole script takes like 30 seconds. You end up with these files:

```
tree ~/.m2/repository/org/elasticsearch/elasticsearch-plugin-classloader/7.11.0
/home/alex/.m2/repository/org/elasticsearch/elasticsearch-plugin-classloader/7.11.0
├── elasticsearch-plugin-classloader-7.11.0.jar
├── elasticsearch-plugin-classloader-7.11.0.module
└── elasticsearch-plugin-classloader-7.11.0.pom
```

**Comment by rjernst** [comment]
This was resolved by #68889
